polaris-react/src/components/TopBar/components/UserMenu/UserMenu.tsx

```diff
@@ -34,6 +34,8 @@ export interface UserMenuProps {
   customActivator?: React.ReactNode;
   /** A width value that customizes the width of the user menu */
   customWidth?: string;
+  /** Whether to indent the menu items, or not */
+  indent?: boolean;
```

Comment:
It's a bit strange to set it here since it seems like it would apply to the whole menu and not just one section of it.
Perhaps we are better of specifying it as part of the actions.
My suggestion would be to modify the `actions`  prop to something unique to `UserMenu`. Maybe something like this:

1. Update ActionList item to allow for a `readOnly` prop so that we can render anytime as non-interactive

2. change to `actions` prop here to something that overrides `ActionListSection` and add a prop for indent:

```typescript
interface UserMenuAction extends ActionListSection {
    indent?: boolean;
}

actions: UserMenuAction[]
```

Reply:
I guess we need at least one item that isn't indented at the bottom. In that case perhaps we can include the `indent` per item. Or we can have "unseparated" sections to make it easer:
```typescript
interface UserMenuAction extends ActionListSection {
    indent?: boolean;
    seamless?: boolean;
}
```

basically it would be used like:
```typescript
[
  {
    name: 'store-list',
    indent: true,
    items: [
      {
        name: 'current shop',
        readOnly: true,
        active, suffix, etc
      },
      ...otherItems,
    ]
  } ,
  {
    name: 'switch-org',
    seamless: true,
    items: [ switcheOrgButton]
  },
  ...otherSections
]
```

Reply:
I like the wrapper having to indent the section rather than individual items. Let me know what you think in the new approach